src/appkit.cs

```diff
@@ -19275,8 +19275,14 @@ partial interface NSTextView : NSTextInputClient, NSTextLayoutOrientationProvide
 		[Export ("toggleSmartInsertDelete:")]
 		void ToggleSmartInsertDelete (NSObject sender);
 
-		[Export ("smartInsertForString:replacingRange:beforeString:afterString:")]
```

Comment:
This will be a binary breaking change because the `virtual` modifier disappeared from this overload.

Reply:
@spouliot ? Did I mess up applying your [suggestion](https://github.com/xamarin/xamarin-macios/pull/8365#discussion_r408132203) or do i need to roll it back?

Reply:
no, just add `, IsVirtual = true` to the `[Wrap]` attribute
and double check with apidiff